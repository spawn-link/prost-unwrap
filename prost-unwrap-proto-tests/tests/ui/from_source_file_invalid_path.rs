mod test {
    // NB: trybuild alters the cwd, so the adjustment is needed; actual cwd
    // within trybuild test is
    // $WORKSPACE/target/tests/trybuild/prost-unwrap-proto-tests
    prost_unwrap::include!(from_source(root, InvalidSourceFileArgType));
}

fn main() {}
