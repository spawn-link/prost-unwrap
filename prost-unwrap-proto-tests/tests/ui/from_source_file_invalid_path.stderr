error: Parameter argument must be a string literal path to the prost-generated rust source code
 --> tests/ui/from_source_file_invalid_path.rs:5:46
  |
5 |     prost_unwrap::include!(from_source(root, InvalidSourceFileArgType));
  |                                              ^^^^^^^^^^^^^^^^^^^^^^^^
